LifeCycleAssessment_Engine/Compute/EvaluateEnvironmentalProductDeclarationByLength.cs

```diff
@@ -44,7 +45,7 @@ public static partial class Compute
         [Input("field", "Filter the provided EnvironmentalProductDeclaration by selecting one of the provided metrics for calculation.")]
         [Input("exactMatch", "If true, the evaluation method will force an exact LCA phase match to solve for.")]
         [Output("quantity", "The total quantity of the desired metric based on the EnvironmentalProductDeclarationField.")]
-        private static EnvironmentalMetricResult EvaluateEnvironmentalProductDeclarationByLength(IElementM elementM, List<LifeCycleAssessmentPhases> phases, EnvironmentalProductDeclarationField field = EnvironmentalProductDeclarationField.GlobalWarmingPotential, bool exactMatch = false)
+        private static EnvironmentalMetricResult EvaluateEnvironmentalProductDeclarationByLength(IElementM elementM, List<LifeCycleAssessmentPhases> phases, MaterialComposition materialComposition, EnvironmentalProductDeclarationField field = EnvironmentalProductDeclarationField.GlobalWarmingPotential, bool exactMatch = false)
         {
             if (elementM is IElement1D)
             {
```

Comment:
_comment actually refers to line 54._
This should be updated to call to the new GetEvaluationValue method with MaterialComposition included to eliminate the redundant MaterialComposition call within the currently called method.